wcomponents-core/src/main/java/com/github/bordertech/wcomponents/WText.java

```diff
@@ -78,6 +80,25 @@ public void setText(final String text, final Serializable... args) {
 	}
 

```

Comment:
I am concerned the direction we are sanitizing (ie on the get or set). Many times WText is used with a "bean property" and "setData" is never called.

Maybe we should do the sanitizing on getData?